Cargo.toml

```diff
@@ -21,6 +21,7 @@ indicatif = "0.9.0"
 lazy_static = "1.1.0"
 openssl = { version = '0.10.11', optional = true }
 parking_lot = "0.6"
+path-absolutize = "1.1.1"
```

Comment:
Was there a particular reason for using `path-absolutize`, which implements its own custom logic, rather than just calling `GetFullPathName` on Windows? I expect the latter would be more robust.